app.js

```diff
@@ -18,98 +22,36 @@ var allowCrossDomain = function(req, res, next) {
     if ('OPTIONS' == req.method) {
       res.send(200);
     }
-    else {
-      next();
-    }
+    else { next(); }
```

Comment:
why did you oneline just the else? either make this a ternary or leave it be :dancer: